src/collatz.js

```diff
@@ -0,0 +1,17 @@
+  export default function collatz(n){
+
+    var anArray = []
+    anArray.push(n)
+    
+    while(n > 1){
+      if(n % 2 !== 0){
+        n = n * 3 + 1
+        anArray.push(n)
```

Comment:
Extract this line out of both the `if` and `else` paths of the conditional and place it at the end of the `while` loop ([DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself) principle).